packages/story-editor/src/components/form/hierarchical/index.js

```diff
@@ -116,7 +116,7 @@ const Option = (option) => {
         <Label htmlFor={optionId}>{label}</Label>
       </CheckboxContainer>
       {options?.map((child) => (
-        <StepContainer key={`${child.id}-${child.checked}`}>
```

Comment:
unfortunately this breaks a karma test: https://github.com/google/web-stories-wp/pull/9199. The checkboxes lose focused when pressed because they get re-created entirely.